azure-pipelines.yml

```diff
@@ -32,42 +32,6 @@ stages:
       - bash: echo "##vso[task.setvariable variable=start_main;isOutput=true]$RET"
         name: result
 
-- stage: InitialTests
-  condition: and(succeeded(), eq(dependencies.Check.outputs['Skip.result.start_main'], 'true'))
-  dependsOn: Check
-  jobs:
-
-  # Native build is based on gcc flag `-march=native`
-  - job: Linux_baseline_native
-    pool:
-      vmImage: 'ubuntu-20.04'
-    steps:
-    - script: |
-        git submodule update --init
-      displayName: 'Fetch submodules'
-    - script: |
-            if ! `gcc 2>/dev/null`; then
-                sudo apt install gcc
-            fi
-            sudo add-apt-repository ppa:deadsnakes/ppa -y
-            sudo apt install python3.9
-            sudo apt install python3.9-dev
-            sudo apt install python3.9-distutils
-            # python3 has no setuptools, so install one to get us going
-            python3.9 -m pip install --user --upgrade pip 'setuptools<49.2.0'
-            python3.9 -m pip install --user -r test_requirements.txt
-      displayName: 'install python/requirements'
-    - script: |
-            python3.9 runtests.py --show-build-log --cpu-baseline=native --cpu-dispatch=none \
-            --debug-info --mode=full -- -rsx --junitxml=junit/test-results.xml
-      displayName: 'Run native baseline Build / Tests'
-    - task: PublishTestResults@2
-      condition: succeededOrFailed()
-      inputs:
-        testResultsFiles: '**/test-*.xml'
-        failTaskOnFailedTests: true
-        testRunTitle: 'Publish test results for baseline/native'
-
```

Comment:
Delete this build job, we have a python3.9 linux job in the github build_test action.